src/MemberProfile.js

```diff
@@ -1,17 +1,115 @@
 import React from "react";
 import { Link } from "react-router-dom";
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import Chip from 'material-ui/Chip';
+import FontIcon from 'material-ui/FontIcon';
+import AppBar from 'material-ui/AppBar';
+import SvgIconFace from 'material-ui/svg-icons/action/face';
 
 const MemberProfile = ({ user }) => {
   return (
-    <div>
-      <img src={user.picture} alt="avatar" />
-      <h1>{user.name}</h1>
-      <p>{user.description}</p>
-      <p>{user.email}</p>
-      <p>GitHub: {user.githubName}</p>
-      <p>Flowdock: {user.flowdockName}</p>
-      <Link to="/me/edit">edit</Link>
+  <MuiThemeProvider>
+    <div className="container">
+      <div className="card">
+        <div className="pic">
+          <div className="container-nested">
+            <img src="http://placehold.it/225x225"/>
+          </div>
+        </div>
+        <div className="user-bio">
+          <div className="container-nested">
+            <h2>FirstName LastName</h2>
+            <p>"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut 
+            labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris 
+            nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit 
+            esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in 
+            culpa qui officia deserunt mollit anim id est laborum."</p>
+            <a href="mailto:email@gmail.com">email@gmail.com</a>
+              <div className= "contact">
+                <span>GitHub: </span><a href="github.com">github.com</a>
```

Comment:
It looks like we also have the github name based on the code above this would be:
```jsx
<span>GitHub: </span><a href={`github.com/${user.githubName}`}>{user.githubName}</a>
```
Also do we need the span around the `GitHub` label? For that matter, I suppose an html `label` element would actually be more semantic if anything.